recipes/pyophidia/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: pyophidia
+    version: "1.4.0"
+
+source:
+    url: git://github.com/OphidiaBigData/PyOphidia/archives/v1.4.0.tar.gz
+    sha256: b276b11d8e72396f00a95eeec0086df75e6fe6980106eb2fab68d237c665a1f7
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - PyOphidia
+
+about:
+    home:  http://github.com/OphidiaBigData/PyOphidia
+    license:  GPLv3
+    licence_file: LICENSE
+    summary: PyOphidia is a Python package for interacting with the Ophidia framework
+
+extra:
+    recipe-maintainers:
+        - doutriaux1
+        - eldoo
```

Comment:
@eldoo, @minomicetto, @aledanca, @sandrofiore -- Are you ok being added as co-maintainers of the `PyOphidia` package on conda-forge?

Reply:
Yes, it is fine with me.

Reply:
Yes, it's ok with me.

Reply:
Yes, of course

Reply:
Yes, sure.